marklogic-data-hub-central/ui/src/components/load-data/new-data-load-dialog/new-data-load-dialog.tsx

```diff
@@ -14,20 +14,17 @@ const NewDataLoadDialog = (props) => {
   const [inputFilePath, setInputFilePath] = useState(props.stepData && props.stepData.inputFilePath ? props.stepData.inputFilePath : '');
   const [srcFormat, setSrcFormat] = useState(props.stepData && props.stepData != {} ? props.stepData.sourceFormat : 'json');
   const [tgtFormat, setTgtFormat] = useState(props.stepData && props.stepData != {} ? props.stepData.targetFormat : 'json');
-  const [outUriReplacement, setOutUriReplacement] = useState(props.stepData && props.stepData != {} ? props.stepData.outputURIReplacement : '');
+  const [outUriPrefix, setOutUriPrefix] = useState(props.stepData && props.stepData != {} ? props.stepData.outputURIPrefix : '');
```

Comment:
Can this be setOutputUriPrefix instead of setOutUriPrefix? I am generally against any sort of abbreviations like this - same goes for "src" and "tgt" instead of "source" and "target". If this were a local variable in a function that's just a few lines long, I think extremely short abbreviations can work. But otherwise, we should use regular words, no need for abbreviations that can potentially confuse.